rage/i18n/ru/rage.ftl

```diff
@@ -56,454 +56,460 @@
+    ожно указать несколько идентификаторов, неиспользуемые будут игнорироваться.
+    {-stdin} может использоваться для чтения идентификаторов со стандартного ввода.
 
 rage-after-help-example =
-    Example:
+    Пример:
     {"  "}{$example_a}
     {"  "}{tty-pubkey}: {$example_a_output}
     {"  "}{$example_b}
     {"  "}{$example_c}
 
-keygen-help-flag-output = {help-flag-output} Defaults to standard output.
-keygen-help-flag-convert = Convert an identity file to a recipients file.
+keygen-help-flag-output = {help-flag-output} По умолчанию использует стандартный вывод.
+keygen-help-flag-convert = Преобразовать файл идентификации в файл получателей.
 
 ## Formatting
 
-warning-msg = Warning: {$warning}
+warning-msg = Внимание: {$warning}
 
 ## Keygen messages
 
-tty-pubkey = Public key
-identity-file-created = created
-identity-file-pubkey = public key
+tty-pubkey = Публичный ключ
+identity-file-created = создан
+identity-file-pubkey = публичный ключ
 
 ## Encryption messages
 
-autogenerated-passphrase = Using an autogenerated passphrase:
-type-passphrase = Type passphrase
-prompt-passphrase = Passphrase
+autogenerated-passphrase = Использование автоматически сгенерированного пароля:
+type-passphrase = Введите пароль
+prompt-passphrase = Пароль
 
-warn-double-encrypting = Encrypting an already-encrypted file
+warn-double-encrypting = Шифрование уже зашифрованного файла
 
 ## General errors
 
-err-failed-to-open-input = Failed to open input: {$err}
-err-failed-to-open-output = Failed to open output: {$err}
-err-failed-to-read-input = Failed to read from input: {$err}
-err-failed-to-write-output = Failed to write to output: {$err}
-err-identity-ambiguous = {-flag-identity} requires either {-flag-encrypt} or {-flag-decrypt}.
-err-mixed-encrypt-decrypt = {-flag-encrypt} can't be used with {-flag-decrypt}.
-err-passphrase-timed-out = Timed out waiting for passphrase input.
-err-same-input-and-output = Input and output are the same file '{$filename}'.
-
-err-ux-A = Did {-rage} not do what you expected? Could an error be more useful?
-err-ux-B = Tell us
-# Put (len(A) - len(B) - 32) spaces here.
+err-failed-to-open-input = Не удалось открыть входной файл: {$err}
+err-failed-to-open-output = Не удалось открыть выходной файл: {$err}
+err-failed-to-read-input = Не удалось прочитать из входного файла: {$err}
+err-failed-to-write-output = Не удалось записать в выходной файл: {$err}
+err-identity-ambiguous = {-flag-identity} требует либо {-flag-encrypt}, либо {-flag-decrypt}.
+err-mixed-encrypt-decrypt = {-flag-encrypt} не может использоваться вместе с {-flag-decrypt}.
+err-passphrase-timed-out = Истекло время ожидания ввода пароля.
+err-same-input-and-output = Входной и выходной файлы совпадают '{$filename}'.
+
+err-ux-A = {-rage} не сделал то, что вы ожидали? Могла ли быть полезнее ошибка?
+err-ux-B = Сообщите нам
+# Поставьте здесь пробелы (len(A) - len(B) - 32).
 err-ux-C = {"                          "}
```

Comment:
For reference, you don't need to translate the comment (though it is fine to do so for the benefit of future `ru` translators); it is there to tell you as the translator what to do with this string. In this case, `err-ux-A` has length 65 (after substituting `{-rage}`), and `err-ux-B` has length 12, so `err-ux-C` should contain 21 space characters.